android/titanium/src/java/org/appcelerator/titanium/proxy/TiWindowProxy.java

```diff
@@ -568,6 +568,9 @@ protected void fillIntent(Activity activity, Intent intent)
 			// We're opening child activity from Titanium root activity. Have it exit out of app by default.
 			// Note: If launched via startActivityForResult(), then root activity won't be the task's root.
 			intent.putExtra(TiC.INTENT_PROPERTY_FINISH_ROOT, true);
+
+			// Set default value as stated in docs on first window also if not already set above.
+			setProperty(TiC.PROPERTY_EXIT_ON_CLOSE, true);
```

Comment:
are you sure this belongs into this PR?

Reply:
I think it slipped into this PR as it was done on the other PR we were testing out related to base activity. Thanks for the catch!